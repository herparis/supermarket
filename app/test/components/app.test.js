import React from 'react'
import { shallow } from 'enzyme'

import {
  Title, ProductsManager, AddModal,
} from '../../src/components'
import App from '../../app'

function checkRender(wrapper) {
  expect(wrapper.isEmptyRender()).toBe(false)
  expect(wrapper).toMatchSnapshot()
}

describe('App components rendeding', () => {
  const app = shallow(<App />)
  it('AppWrapper renders', () => {
    checkRender(app)
  })
  it('Title renders', () => {
    const title = app.find(Title)
    checkRender(title)
  })
  it('ProductsManager renders', () => {
    const productsManager = app.find(ProductsManager)
    checkRender(productsManager)
  })
  it('AddModal renders', () => {
    const addModal = app.find(AddModal)
    checkRender(addModal)
  })
})
