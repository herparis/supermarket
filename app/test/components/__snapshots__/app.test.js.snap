// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App components rendeding AddModal renders 1`] = `ShallowWrapper {}`;

exports[`App components rendeding AppWrapper renders 1`] = `ShallowWrapper {}`;

exports[`App components rendeding ProductsManager renders 1`] = `ShallowWrapper {}`;

exports[`App components rendeding Title renders 1`] = `ShallowWrapper {}`;
